src/main/kotlin/view/OutputView.kt

```diff
@@ -0,0 +1,30 @@
+package view
+
+import domain.Car
+import domain.Cars
+
+private const val COLON = " : "
+private const val DASH = "-"
+
+object OutputView {
+
+    fun printException(e: Exception) {
+        println(e.message)
+        println()
+    }
+
+    fun printResultTitle() {
+        println("실행 결과")
+    }
+
+    fun printRaceResult(cars: Cars) {
+        cars.value.forEach {
+            println(it.name + COLON + DASH.repeat(it.position))
```

Comment:
```suggestion
        for (car in cars.value) {
            println(car.name + COLON + DASH.repeat(car.position))
```
제 경우는 이런 상황에서 forEach보다 일반for문을 사용하는 것이 잘 읽히더라고요. 스타일 차이니까 팀 그라운드에 맞춰 팀원들과 조율해가면 좋을 부분이네요.
(위대한 머쉬맘 말씀대로 문자열 템플릿을 사용하는 것이 좋아보여요)